pkg/operator/sync.go

```diff
@@ -1359,6 +1361,12 @@ func (optr *Operator) syncMachineConfigDaemon(config *renderConfig) error {
 		configMaps: []string{
 			mcdKubeRbacProxyConfigMapPath,
 		},
+		validatingAdmissionPolicies: []string{
```

Comment:
probably a dumb question, what is the mechanism by which this is being gated by featuregate?

As in, this will always sync, but if we dont have techpreview on, it will see the object as `apiVersion: admissionregistration.k8s.io/v1beta1` and refuse to apply it...?

Reply:
It's currently gated in applyManifests [here](https://github.com/openshift/machine-config-operator/blob/aa28d1b034bcfaff79537d2fc979de018a77e4a0/pkg/operator/sync.go#L902). VAP does GA in 1.30, so we'll have to update this to respect the actual features at that point. If we potentially time the kube bump with MCN GAing we could skip that middle step